qmlti/lti/LTI_Tool_Provider.php

```diff
@@ -360,7 +365,11 @@ private function result() {
       } else if (!is_null($this->redirectURL)) {
         header("Location: {$this->redirectURL}");
       } else if (!is_null($this->output)) {
-        echo $this->output;
+        if ($this->output == "student.php" || $this->output == "staff.php") {
+          header("Location: {$this->output}");
+        } else {
+          echo $this->output;
```

Comment:
I don't know what the echo was initially used for, nor can I find a way to make it useful. I've removed it since, thanks for pointing this out!